utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+            Number of rows in the table
+        """
+        try:
+            cursor = self.connection.cursor()
+            count_query = sql.SQL("SELECT COUNT(*) AS cnt FROM {}.{}").format(
+                sql.Identifier(schema), sql.Identifier(table_name)
+            )
+            cursor.execute(count_query)
+            row = cursor.fetchone()
+            count = row.get('cnt') if isinstance(row, dict) else row[0]
+            cursor.close()
+            return count
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)